core/src/test/java/org/apache/calcite/test/SqlJsonFunctionsTest.java

```diff
@@ -534,13 +534,26 @@ public void testJsonKeys() {
   @Test
   public void testJsonRemove() {
     assertJsonRemove(
-        JsonFunctions.jsonValueExpression("{\"a\": 1, \"b\": [2]}"),
-        new String[]{"$.a"},
-        is("{\"b\":[2]}"));
+            JsonFunctions.jsonValueExpression("{\"a\": 1, \"b\": [2]}"),
```

Comment:
I am not sure if our checkstyle checkers works appropriately in this case, but we actually use the number 4 for continuation indent. Please adjust accordingly.